api/src/services/occurrence-service.ts

```diff
@@ -34,6 +57,140 @@ export class OccurrenceService extends DBService {
     this.occurrenceRepository = new OccurrenceRepository(connection);
   }
 
+  /**
+   * Get all occurrences within map bounds
+   *
+   * @param {(string | undefined)} [mapView]
+   * @return {*}  {Promise<IGetMapOccurrenceData[]>}
+   * @memberof OccurrenceService
+   */
+  async getMapOccurrences(mapView?: string | undefined): Promise<IGetMapOccurrenceData[]> {
+    const allOccurrences = await this.occurrenceRepository.getMapOccurrences(mapView);
+
+    const formated = this.formatOccurrenceDataForMap(allOccurrences);
```

Comment:
super nit: maybe change `formated` to formattedOccurrences` just to be clear that its still occurrences (as minor as that is)/